.github/actions/setup-gke-cluster/action.yml

```
@@ -0,0 +1,74 @@
+name: "Create GKE cluster"
+description: "Uses gcloud"
+inputs:
+  cluster-name:
+    description: "Base cluster name"
+    required: true
+  zone:
+    description: "GCP zone for the cluster"
+    required: true
+  cluster-version:
+    description: "Explicit GKE cluster version"
+    required: false
+    default: ""
+  mode:
+    description: ""
+    required: false
+    default: ""
```

```suggestion
  node-taints:
    description: "taints to apply to cluster nodes"
    required: false
    default: "ignore-taint.cluster-autoscaler.kubernetes.io/cilium-agent-not-ready=true:NoExecute"
```
This `mode` input relies on implementation details (the magic value `baseline`).

What if we change the input to specify the node taints, and supply `node-taints: ""` in cases we do not want the default (i.e. matrix.mode=baseline)?

---

yep, agree with this input, I've implemented it, please lmk if it's good for you